packages/e2e-selenium/README.md

```diff
@@ -0,0 +1,13 @@
+# E2E Selenium
+
+End-To-End (E2E) tests for WordPress written for legacy browsers.
```

Comment:
I guess `legacy` is a bit wrong here, this package might test edge cases in safari, firefox, or edge. Right?

Reply:
> I guess `legacy` is a bit wrong here, this package might test edge cases in safari, firefox, or edge. Right?

Related to https://github.com/WordPress/gutenberg/pull/21528#issuecomment-612888456 , I think it might depend on the scope of the issues we want to cover here. If we don't think we'll need it to be used for Firefox and Safari, then I think it's quite reasonable to document this as targeted toward legacy browsers. In fact, I think it could be _preferable_ to further clarify the limited scope this package is intended to serve.

Reply:
But this change breaks compressed size test: https://github.com/WordPress/gutenberg/pull/21528/checks?check_run_id=584478893

Reply:
I would be perfectly happy to have these tests running on IE only. None of the other browsers have caused (or are likely to cause) anywhere near as much trouble as IE.